src/components/JdsButton/Button.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <button 
+  :type="type"
```

Comment:
i think better for make `type` attribute passed into child component, remove `type` property in `JDSButton` component

Reply:
I agree, changes [here](https://github.com/jabardigitalservice/jds-design-system-vue-lib/pull/133/commits/6bd20661937cd70526f9db138762646a3ad19679). Thank you!